src/dom/index.js

```diff
@@ -73,7 +74,12 @@ export function setAccessor(node, name, old, value, isSvg) {
 		(node._listeners || (node._listeners = {}))[name] = value;
 	}
 	else if (name!=='list' && name!=='type' && !isSvg && name in node) {
-		setProperty(node, name, value==null ? '' : value);
+        /** Attempt to set a DOM property to the given value.
+         *	IE & FF throw for certain property-value combinations.
+         */
+		try {
```

Comment:
Moving this inline deopts setAccessor(), which is called a lot. It was in a separate function so that it would only deopt when called.